ietf/doc/templatetags/ietf_filters.py

```diff
@@ -1022,7 +1022,7 @@ def is_in_stream(doc):
 def can_issue_ietf_call_for_adoption(doc):
     return all(
         [
-            doc.stream_id in ["ietf", None],
```

Comment:
Isn't the `doc.stream_id is None` case needed to allow calls for adopting individual submissions?